buildflow/core/options/flow_options.py

```diff
@@ -1,18 +1,67 @@
 import dataclasses
 
 from buildflow.core.options._options import Options
-from buildflow.core.options.infra_options import InfraOptions
-from buildflow.core.options.runtime_options import RuntimeOptions
+from buildflow.core.options.infra_options import InfraOptions, PulumiOptions
+from buildflow.core.options.runtime_options import AutoscalerOptions, RuntimeOptions
 
 
 @dataclasses.dataclass
 class FlowOptions(Options):
```

Comment:
@JoshTanke think these are fine in flow options or show we flatten these as options to `Flow` directly?

Reply:
I like it in flow options. Maybe a general rule of thumb is we only flatten options if the method contains a single options type. I.e. we can set processor/pipeline level options in the @pipeline() decorator. So maybe _just_ the top-level flow options could be flattened in Flow? (I dont think we have any yet, but could maybe just follow this pattern when we do)

Reply:
I think we still want to expose autoscale options at the top level though and those are technically nested in runtime